openpower/configs/hostboot/witherspoon.config

```diff
@@ -29,7 +29,7 @@ unset CDIMM_FORMAT_FOR_CVPD
 set GPIODD
 unset PALMETTO_VDDR
 
-# Enable SBE updates
+# Disable SBE updates
```

Comment:
Please put in a separate patch, this has nothing to do with MEMVPD configuration, it's correcting a comment.